# HG changeset patch
# User Jaroslav Hajek <highegg@gmail.com>
# Date 1265814930 -3600
#      Wed Feb 10 16:15:30 2010 +0100
# Node ID ed88ea0367168fb14ab9090d2521c553f3edc6c9
# Parent  035ac548a67e03ade5499ea87dfd0ad20c2231d9
improve docs of fzero/fminbnd

diff --git a/scripts/ChangeLog b/scripts/ChangeLog
--- a/scripts/ChangeLog
+++ b/scripts/ChangeLog
@@ -1,8 +1,14 @@
+2010-02-10  Jaroslav Hajek  <highegg@gmail.com>
+
+	* optimization/fminbnd.m: Undocument impossible info values.
+	* optimization/fzero.m: Ditto.
+
+	* optimization/fzero.m: Make default TolX consistent with fminbnd.
 2010-02-10  Jaroslav Hajek  <highegg@gmail.com>
 
 	* optimization/fminbnd.m: New function.
 	* optimization/module.mk: Add it here.
 	* optimization/fzero.m: Make default TolX consistent with fminbnd.
 
 2010-02-10  Jaroslav Hajek  <highegg@gmail.com>
 
diff --git a/scripts/optimization/fminbnd.m b/scripts/optimization/fminbnd.m
--- a/scripts/optimization/fminbnd.m
+++ b/scripts/optimization/fminbnd.m
@@ -25,29 +25,24 @@
 ## structure specifying additional options.  Currently, @code{fminbnd}
 ## recognizes these options: @code{"FunValCheck"}, @code{"OutputFcn"},
 ## @code{"TolX"}, @code{"MaxIter"}, @code{"MaxFunEvals"}. 
 ## For description of these options, see @ref{doc-optimset,,optimset}.
 ## 
 ## On exit, the function returns @var{x}, the approximate minimum point
 ## and @var{fval}, the function value thereof.
 ## @var{info} is an exit flag that can have these values:
+##
 ## @itemize
 ## @item 1
 ## The algorithm converged to a solution.
 ## @item 0
 ## Maximum number of iterations or function evaluations has been exhausted.
 ## @item -1
 ## The algorithm has been terminated from user output function.
-## @item -2 
-## A general unexpected error.
-## @item -3
-## A non-real value encountered.
-## @item -4
-## A NaN value encountered.
 ## @end itemize
 ## @seealso{optimset, fzero, fminunc} 
 ## @end deftypefn
 
 ## This is patterned after opt/fmin.f from Netlib, which in turn is taken from
 ## Richard Brent: Algorithms For Minimization Without Derivatives, Prentice-Hall (1973)
 
 ## PKG_ADD: __all_opts__ ("fminbnd");
diff --git a/scripts/optimization/fzero.m b/scripts/optimization/fzero.m
--- a/scripts/optimization/fzero.m
+++ b/scripts/optimization/fzero.m
@@ -25,29 +25,24 @@
 ## structure specifying additional options.  Currently, @code{fzero}
 ## recognizes these options: @code{"FunValCheck"}, @code{"OutputFcn"},
 ## @code{"TolX"}, @code{"MaxIter"}, @code{"MaxFunEvals"}. 
 ## For description of these options, see @ref{doc-optimset,,optimset}.
 ## 
 ## On exit, the function returns @var{x}, the approximate zero point
 ## and @var{fval}, the function value thereof.
 ## @var{info} is an exit flag that can have these values:
+##
 ## @itemize
 ## @item 1
 ## The algorithm converged to a solution.
 ## @item 0
 ## Maximum number of iterations or function evaluations has been exhausted.
 ## @item -1
 ## The algorithm has been terminated from user output function.
-## @item -2 
-## A general unexpected error.
-## @item -3
-## A non-real value encountered.
-## @item -4
-## A NaN value encountered.
 ## @end itemize
 ## @seealso{optimset, fsolve} 
 ## @end deftypefn
 
 ## This is essentially the ACM algorithm 748: Enclosing Zeros of
 ## Continuous Functions due to Alefeld, Potra and Shi, ACM Transactions
 ## on Mathematical Software, Vol. 21, No. 3, September 1995. Although
 ## the workflow should be the same, the structure of the algorithm has
